configs/en-lt-spring-2024.yml

```diff
@@ -0,0 +1,195 @@
+# The initial configuration was generated using:
+# task config-generator -- en lt --name spring-2024
+#
+# The documentation for this config can be found here:
+# https://github.com/mozilla/firefox-translations-training/blob/001d15d5f2775e0e4c57717057a1906069e29dcc/taskcluster/configs/config.prod.yml
+experiment:
+  name: spring-2024
+  src: en
+  trg: lt
+  best-model: chrf
+  use-opuscleaner: 'true'
+  opuscleaner-mode: defaults
+  bicleaner:
+    default-threshold: 0.5
+    dataset-thresholds: {}
+  mono-max-sentences-src: 500_000_000
+  mono-max-sentences-trg: 200_000_000
+  spm-sample-size: 10_000_000
+  spm-vocab-size: 32000
+  teacher-ensemble: 2
+  teacher-mode: two-stage
+  pretrained-models: {}
+datasets:
+
+  # Skipped test/devtest datasets:
+  #  - wmt19/dev - variant dataset
+  devtest:
+  - flores_aug-mix_dev
```

Comment:
I had to add `  - sacrebleu_aug-mix_wmt19/dev` here manually. Let's output everything from `find-cropus` that's being skipped.
Here's my config:
```
  devtest:
  - flores_aug-mix_dev
  - sacrebleu_aug-mix_wmt19/dev
  - mtdata_aug-mix_Neulab-tedtalks_dev-1-eng-lit

  test:
  - flores_devtest
  - flores_aug-mix_devtest
  - flores_aug-title_devtest
  - flores_aug-upper_devtest
  - flores_aug-typos_devtest
  - flores_aug-noise_devtest
  - flores_aug-inline-noise_devtest
  - sacrebleu_wmt19
  - mtdata_Neulab-tedtalks_test-1-eng-lit
```